added isEU (#60)
ipinfo/node: README.md

### Country Name Lookup

> `response.country` will return the country name, whereas `response.countryCode` can be used to fetch country code and response.isEU will return true if the country is a member of EU.

Reviewer: ```suggestion
`response.country` will return the country name, whereas `response.countryCode` can be used to fetch country code.

Additionally `response.isEU` will return true if the country is a member of European Union (EU).
```

    // country name, e.g. 'United States'
    console.log(response.country);

>     // false 

Reviewer: ```suggestion
    // whether part of the EU, e.g. false
```

    // country name, e.g. 'United States'
    console.log(response.country);

>     // false 

Reviewer: ```suggestion
    // whether part of the EU, e.g. false
```
